### DISCUSSION TITLE
No response from support

### DISCUSSION THREAD

**Discussion by ZehMatt** [community/community#48647] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've been waiting for 18 days now and there is no response not even from a bot. Considering what my ticket is about it should be something that one of the support staff can do in a couple of clicks. My last contact with the support was pretty painless and quick, are there really that many tickets generated each day now that I would not get a response at least within a week or two?

**Comment by sbhask** [comment]
You could try tweeting at the CEO

https://twitter.com/ashtom

**Comment by Hercztonic** [comment]
@ZehMatt Support™ is not responding same here.
